README.md

```diff
@@ -68,7 +68,7 @@ The **Leaderboard List App** project is aimed at helping developers enhance thei
 
 <!-- Live Demo -->
 # 🚀 Live Demo <a name="live-demo"></a>
-To access the Live Demo [Click here!](https://graujavier.github.io/leaderboard/)
+To access the Live Demo [Click here!](https://graujavier.github.io/leaderboard/dist/)
```

Comment:
_Well done 🥇🥇 Live demo link is working. Keep up the great work_ 

![image](https://github.com/grauJavier/leaderboard/assets/106119105/d6b394f2-b3a2-4bd8-ba2c-2596cee9746b)